full-node/sov-ethereum/src/lib.rs

```diff
@@ -174,14 +177,46 @@ pub mod experimental {
             },
         )?;
 
+        #[cfg(feature = "local")]
         rpc.register_async_method("eth_accounts", |_parameters, ethereum| async move {
             Ok::<_, ErrorObjectOwned>(ethereum.eth_rpc_config.eth_signer.signers())
         })?;
 
-        // TODO https://github.com/Sovereign-Labs/sovereign-sdk/issues/502
-        rpc.register_async_method("eth_sendTransaction", |parameters, _ethereum| async move {
-            let _data: reth_rpc_types::TransactionRequest = parameters.one().unwrap();
-            unimplemented!("eth_sendTransaction not implemented")
+        #[cfg(feature = "local")]
+        rpc.register_async_method("eth_sendTransaction", |parameters, ethereum| async move {
+            let typed_transaction: TypedTransaction = parameters.one().unwrap();
+
+            let signed_tx = ethereum
+                .eth_rpc_config
+                .eth_signer
+                .sign_ethers_transaction(typed_transaction)
```

Comment:
There is a similar logic in reth for tx signing: https://github.com/paradigmxyz/reth/blob/main/crates/rpc/rpc/src/eth/signer.rs#L74. The `reth:DevSigner` is not exported so we can't use it but we should be able to copy the relevant logic to our signer and use reth types instead of ethers.